chat-commands.js

```diff
@@ -2921,13 +2921,17 @@ exports.commands = {
 		if (!user.hasConsoleAccess(connection)) {
 			return this.errorReply("/bash - Access denied.");
 		}
-
+		if (!target) {
+			return this.parse('/help bash');
+		}
 		connection.sendTo(room, "$ " + target);
 		let exec = require('child_process').exec;
 		exec(target, (error, stdout, stderr) => {
 			connection.sendTo(room, ("" + stdout + stderr));
 		});
 	},
+	bashhelp:["/bash [argument] - Carries out execution by running a child process",
+	 	  "Requires console access."],
```

Comment:
hmm alright, maybe i should also specify what console access means while im at it, I think a brief description of (~, Config.consoleips) would be quite useful right?